package.json

```diff
@@ -320,6 +320,11 @@
                                     "default": []
                                 }
                             }
+                        },
+                        "when": {
+                            "type": "string",
+                            "markdownDescription": "%configuration.java.test.config.when.description%",
+                            "default": "testItem ~= //"
```

Comment:
How about just leave the default value empty `""`

Reply:
https://github.com/microsoft/vscode-java-test/pull/1638/commits/338c10dba870cf61cc170143f7b6008d8a0c41b9